### ISSUE TITLE
Medial collateral ligament (MCL) tear at the elbow

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10798]
**Preferred term label:**
Medial collateral ligament (MCL) tear at the elbow

**Synonyms**

**Definition (free text, please give PubMed ID)**
 ligamentous tear on the medial aspect of the elbow

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Radial head fracture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001133
- label: Elbow medial collateral ligament tear
- parents: [Abnormality of the elbow (HP:0009811)](https://hpo.jax.org/app/browse/term/HP:0009811)
- pmid: PMID:19468906
- def: An injury to the medial collateral ligament (MCL) of the elbow.
- comment: Elbow medial collateral ligament sprain occurs when the elbow is subjected to a valgus force exceeding the tensile properties of the medial collateral ligament (MCL). This is an injury seen more often in throwing athletes.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10798
- synonyms: Ligamentous tear on the medial aspect of the elbow(NONE)
